content/en/docs/reference/scheduling/config.md

```diff
@@ -289,7 +291,187 @@ only has one pending pods queue.
+However, individual extension points are still available to allow flexible 
+reconfiguration of the default values (such as ordering and Score weights). For 
+example, consider two Score plugins `DefaultScore1` and `DefaultScore2`, each with 
+a weight of `1`. They can be reordered with different weights like so:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: multipoint-scheduler
+    plugins:
+      score:
+        enabled:
+        - name: 'DefaultScore2'
+          weight: 5
+```
+
+In this example, it's unnecessary to specify the plugins in `MultiPoint` explicitly 
+because they are default plugins. And the only plugin specified in `Score` is `DefaultScore2`.
+This is because plugins set through specific extension points will always take precedence 
+over `MultiPoint` plugins. So, this snippet essentially re-orders the two plugins 
+without needing to specify both of them.
+
+The general hierarchy for precedence when configuring `MultiPoint` plugins is as follows:
+1. Specific extension points run first, and their settings override whatever is set elsewhere
+2. Plugins manually configured through `MultiPoint` and their settings
+3. Default plugins and their default settings
+
+To demonstrate the above hierarchy, the following example is based on these plugins:
+|Plugin|Extension Points|
+|---|---|
+|`DefaultQueueSort`|`QueueSort`|
+|`CustomQueueSort`|`QueueSort`|
+|`DefaultPlugin1`|`Score`, `Filter`|
+|`DefaultPlugin2`|`Score`|
+|`CustomPlugin1`|`Score`, `Filter`|
+|`CustomPlugin2`|`Score`, `Filter`|
+
+A valid sample configuration for these plugins would be:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: multipoint-scheduler
+    plugins:
+      multiPoint:
+        enabled:
+        - name: 'CustomQueueSort'
+        - name: 'CustomPlugin1'
+          weight: 3
+        - name: 'CustomPlugin2'
+        disabled:
+        - name: 'DefaultQueueSort'
+      filter:
+        disabled:
+        - name: 'DefaultPlugin1'
+      score:
+        enabled:
+        - name: 'DefaultPlugin2'
+```
+
+Note that there is no error for re-declaring a `MultiPoint` plugin in a specific 
+extension point. The re-declaration is ignored (and logged), as specific extension points 
+take precedence.
+
+Besides keeping most of the config in one spot, this sample does a few things:
+* Enables the custom `queueSort` plugin and disables the default one
+* Enables `CustomPlugin1` and `CustomPlugin2`, which will run first for all of their extension points
+* Disables `DefaultPlugin1`, but only for `filter`
+* Reorders `DefaultPlugin2` to run first in `score` (even before the custom plugins)
+
+In older versions of the config, without `multiPoint`, the above snippet would equate to this:
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
```

Comment:
I think this either should be v1beta2 or just add the disable `*`.

Reply:
went with `v1beta2` and pointed that out. `multiPoint.disabled: *` would mean all of the other default plugins need to be explicitly enabled which isn't exactly the comparison I was trying to draw here